homeassistant/components/hue/light.py

```diff
@@ -312,7 +312,9 @@ def effect(self):
     @property
     def effect_list(self):
         """Return the list of supported effects."""
-        return [EFFECT_COLORLOOP, EFFECT_RANDOM]
+        if self.is_philips:
```

Comment:
I don't see why we would do this.

Reply:
Since it works on non-philips bulbs.

I feel like your PR is very Osram focused, as if there are no other non-Philips bulbs 🙈

Reply:
As @awarecan referenced in the issues #8905 and #6122 older OSRAM and Innr bulbs cannot handle command['effect'] = 'none'.
My assumtion was, if you cannot set effect = 'none' the you should not be able to set effect ='colorloop'.

Reply:
OsramLightfy only allows EFFECT_RANDOM for RGB bulbs.
i switched the implementation to only exclude it for osram bulbs